Lab 1/README.md

```diff
@@ -2,18 +2,28 @@
 
 Learn how to push an image of an application to IBM Cloud Container Registry and deploy a basic application to a cluster.
 
-
-# 1. Push an image to IBM Cloud Container Registry
+# 0. Install Prerequisite CLIs and Provision a Kubernetes Cluster
```

Comment:
I think we should separate out this prerequisites into it's own subsection, as it's a bit weird to say, "let's push an image" and then talk about something else first.